src/context/AppContext.tsx

```diff
@@ -19,6 +19,10 @@ type AppContextType = {
   clearCart: () => void;
   selectedMenu: boolean;
   setSelectedMenu: (isOpen: boolean) => void;
+  totalQuantity: number;
+  setTotalQuantity: (number: number) => void;
+  totalCost: number;
+  setTotalCost: (number: number) => void;
```

Comment:
this logic is already in app context
calculateTotalPrice = totalCost
setTotalPrice = setTotalCost

No need to duplicate them